administration/package.json

```diff
@@ -16,6 +16,7 @@
     "@mui/material": "^5.10.9",
     "@zxing/library": "^0.19.1",
     "apollo-upload-client": "^17.0.0",
+    "canonicalize": "^1.0.8",
```

Comment:
This dependency seems unmaintained. Should we just copy its code over as its not much code? https://github.com/erdtman/canonicalize/blob/master/lib/canonicalize.js

Then we can also do a pure TS implementation.

Reply:
tbh, I don't think it's unmaintained, it's just some finished software :D But yeah, I can also copy paste it...

Reply:
(also it has >150k weekly downloads https://www.npmjs.com/package/canonicalize)

Reply:
Replaced it.